features/theme_configuration.feature

```diff
@@ -3,8 +3,11 @@ Feature: Bundling Config file with Theme gems
   I want to be able to pre-configure my gemified theme
   In order to make it easier for other Jekyllites to use my theme
 
-  Scenario: Easy onboarding with a pre-configured theme
-    Given I have a configuration file with "theme" set to "test-theme"
+  Scenario: Enabling import of theme configuration
+    Given I have a configuration file with:
+      | key                 | value      |
+      | theme               | test-theme |
+      | import_theme_config | true       |
```

Comment:
```suggestion
      | ignore_theme_config | true       |
```

But this will break this test. It seems like we'd want to keep this the way it is and write a new test for ignoring the theme configuration.